cat_server/helloworld/unikernel.ml

```diff
@@ -3,6 +3,9 @@ open V1_LWT
 
 module Main (C:CONSOLE) = struct
 
-  let start console = OS.Time.sleep 2.0 >>
+  let start console =
+    OS.Time.sleep 2.0
+
```

Comment:
Most people that have blank lines in their functions either don't know how to desconstruct large functions into named, compact and readable computations units (i.e. don't know how to write readable code) or have a style that is too imperative. Note that other old-timers will also tell you so see e.g. [this message](https://sympa.inria.fr/sympa/arc/caml-list/2015-06/msg00143.html) by @mshinwell.